src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/view/ProjectCreation.kt

```diff
@@ -3,19 +3,27 @@ package org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view
 import de.jensd.fx.glyphs.materialicons.MaterialIcon
 import de.jensd.fx.glyphs.materialicons.MaterialIconView
 import javafx.event.ActionEvent
+import org.wycliffeassociates.otter.jvm.app.ui.chapterpage.view.ProjectPage
+import org.wycliffeassociates.otter.jvm.app.ui.imageLoader
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments.SelectBook
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments.SelectLanguage
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments.SelectResource
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments.SelectAnthology
+import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.viewmodel.ProjectCreationViewModel
+import org.wycliffeassociates.otter.jvm.app.ui.projecthome.ProjectHomeView
 import org.wycliffeassociates.otter.jvm.app.widgets.progressstepper.ProgressStepper
 import tornadofx.*
+import java.io.File
 
 class ProjectCreationWizard: Wizard() {
 
+    val creationViewModel : ProjectCreationViewModel by inject()
     val steps = listOf(MaterialIconView(MaterialIcon.RECORD_VOICE_OVER, "16px"),
-            MaterialIconView(MaterialIcon.COLLECTIONS_BOOKMARK, "16px"),
-            MaterialIconView(MaterialIcon.CROP_SQUARE, "16px"),MaterialIconView(MaterialIcon.BOOK, "16px"))
+            MaterialIconView(MaterialIcon.COLLECTIONS_BOOKMARK, "16px"), imageLoader(File(ClassLoader.getSystemResource("assets/Cross.svg").toURI())),
+          MaterialIconView(MaterialIcon.BOOK, "16px"))
     override  val canGoNext = currentPageComplete
+    override val canFinish= creationViewModel.allPagesComplete
+
     init {
         showStepsHeader = false
         showStepsHeader = false
```

Comment:
Is this a duplicate line?

Reply:
if you're referring to the can coGoNext and can Finish, no, the canGoNext allows you to progress in the wizard. the canFinish activates the finish button once all the pages have been completed by the user.

Reply:
oh i see what you're talking about now, sorry. yeah it is a duplicate